task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -66,34 +64,26 @@ const transformCSV = (csvInput, numOfPracticeTrials, sequentialStimulus) => {
       newRow.distractors = newRow.distractors.map((choice) => camelize(choice));
     }
 
-    totalTrials += 1;
-
-    if (row.notes === 'practice') {
-      totalPractice += 1;
-    }
-
```

Comment:
totalTrials is now updated when the trial is added to stimulusData
totalPractice is unused and so removed